hack/kpng-local-up.sh

```diff
@@ -72,8 +70,18 @@ function install_k8s {
 function install_kpng {
     # substitute it with your changes...
 
+    setup_j2
     echo "Applying template"
-    envsubst <kpng-deployment-ds.yaml.tmpl >kpng-deployment-ds.yaml
+    # Setting vars for generate the kpng deployment based on template
+    kpng_image="${IMAGE}" \
+    image_pull_policy="${PULL}" \
+    backend="${BACKEND}" \
+    config_map_name="${CONFIG_MAP_NAME}" \
+    service_account_name="${SERVICE_ACCOUNT_NAME}" \
+    namespace="${NAMESPACE}" \
+    e2e_backend_args="${BACKEND_ARGS}"\
+    j2 ./kpng-deployment-ds.yaml.j2 -o ./kpng-deployment-ds.yaml
```

Comment:
So I think @qabpea Just removed this? (I had added it originally) I'd like him to take a look as well

Reply:
I'm alright with just using j2 for now though!

Reply:
Is the question where bgp2go is installed?
I did this
    echo "'bpf2go' not found, installing with 'go'"
	# set GOBIN to bin_directory to endure that binary is in search path
	export GOBIN=${install_directory}
	
	#remove GOPATH just to be sure
	export GOPATH=""


The problem I was solving was when a user do not have a GO environment setup, but go is installed and bgp2go is not in the path
go install  installs in the default GOBIN directory which is often not part of the users path.
GOBIN is therfor  set to the local bin dir.

But this should never be triggered unless there is no bgp2go in the path to begin with.

Reply:
It looks like we're not changing that behavior here :)